package.json

```diff
@@ -43,7 +43,7 @@
     "lodash": "^4.6.1",
     "moment": "^2.10.2",
     "riverpig": "^1.1.0",
-    "sqlite3": "^3.1.4",
+    "sequelize": "^3.27.0",
```

Comment:
We depend on `sequelize` anyway, `ILP Kit` uses `sequelize`. I think the only reason for using `knex` was the mssql or oracle support. AFAIK `sequelize` is better.